superset/charts/filters.py

```diff
@@ -21,15 +21,14 @@
 from sqlalchemy.orm.query import Query
 
 from superset import security_manager
+from superset.connectors.sqla.models import SqlaTable
 from superset.models.slice import Slice
 from superset.views.base import BaseFilter
 
 
-class ChartNameOrDescriptionFilter(
-    BaseFilter
-):  # pylint: disable=too-few-public-methods
-    name = _("Name or Description")
-    arg_name = "name_or_description"
+class ChartAllTextFilter(BaseFilter):  # pylint: disable=too-few-public-methods
+    name = _("All Text")
+    arg_name = "all_text_charts"
```

Comment:
Looks like arg_names share a global namespace. We should add the the resource name to other filters so there's no confusion/collision.